### ISSUE TITLE
symfony/phpunit-bridge overwrite some phpunit.xml parameters at runtime

### ISSUE THREAD

**Issue by MichaelDemeyer** [symfony/recipes#476]
bin/phpunit file check following env parameters at runtime :
- SYMFONY_DEPRECATIONS_HELPER
- SYMFONY_PHPUNIT_REMOVE
- SYMFONY_PHPUNIT_VERSION
- SYMFONY_PHPUNIT_DIR
and set a default value if they're not already defined.
Values defined in phpunit.xml are not defined as env values at this point, so default values will apply.

vendor/symfony/phpunit-bridge/bin/simple-phpunit file do not apply phpunit.xml values if env variables are already set.

So basically it is impossible to set the 4 env variables using phpunit.xml

Step to reproduce :
Add 
        `<env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled" force="true"/>`
to phpunit.xml
run bin/phpunit
SYMFONY_DEPRECATIONS_HELPER value will be 999999

**Comment by leevigraham** [comment]
See: https://github.com/symfony/symfony/issues/28726

**Comment by nicolas-grekas** [comment]
FIxed at the recipe level in https://github.com/symfony/recipes/pull/481
setting there vars in the phpunit.xml files is too late. They should be set in .env.test instead (see recipe).
